contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -179,7 +173,27 @@ Finally, run the tests with your custom config:
 make test-e2e-node REMOTE=true IMAGE_CONFIG_FILE="<local file>" [...]
 ```
 
-# Additional Test Options for both Remote and Local execution
+Image configuration files can further influence how FOCUS and SKIP match test cases.
+
+For example:
+
+```sh
+--focus="\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Serial\]"
+```
+
+runs all e2e tests labeled `"[NodeFeature:*]"` and will skip any tests labeled `"[Flaky]"` or `"[Serial]"`.
+
+Two example e2e tests that match this expression are:
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L155
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L175
+
+However, image configuration files select test cases based on the `tests` field.
```

Comment:
nit: let's reword this. Instead of "However, image configuration files select test cases based on the `tests` field" let's mentioned that the `tests` field of the image configuration file can also be used to select test cases.

Reply:
we can handle this nit in a follow-up.